src/utils/DockerUtil.js

```diff
@@ -212,6 +213,7 @@ export default {
   run (name, repository, tag) {
     tag = tag || 'latest';
     let imageName = repository + ':' + tag;
+    name = ContainerStore.generateName(name);
```

Comment:
Ah, good catch! Can we:

- Remove the call to `generateName` in the `ImageCard.react` component?
- Remove the `name` parameter from the `run` action and util function here?